index.html

```diff
@@ -232,17 +232,17 @@ <h1>Get Started</h1>
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </p>
 
-      <form action="/">
+      <form action="https://formspree.io/f/mqkovwwj" method="POST">
         <div class="form">
           <label for="name">Your name</label><br>
-          <input id="name" type="text" placeholder="Donice Ubaru">
+          <input id="name" type="text" name="name" placeholder="Donice Ubaru" maxlength="30" required>
```

Comment:
- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching in your work. Please have a look at the below comparison table to understand the problem in a better way through visualization use the colour `#1A2236` instead of `#000000`, use **16px** font size and use `padding: 25px 0 5px 10px;` for the correct styling.
 
###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-08 06-37-37](https://github.com/donice/Portfolio-Microverse/assets/66476329/a1909c34-bbb0-4204-a24e-040ce1ef181a)

</td>

<td>

![Screenshot from 2023-06-08 06-37-45](https://github.com/donice/Portfolio-Microverse/assets/66476329/bb2d15a3-2c8e-4920-92de-41dc84ee3004)

</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)